### PR TITLE
Update immunization table for two additional fields

### PR DESCRIPTION
**Pull request by burrowse**

### CODE REVIEW

File: pedsnet/v2.8/definitions/immunization.csv [hunk 1]
```diff
@@ -12,3 +12,5 @@ pedsnet,v2.8,immunization,person_id,Yes,person,person_id,A foreign key identifie
 pedsnet,v2.8,immunization,provider_id,No,provider,provider_id,A foreign key to the provider in the provider table who administered the immunization.
 pedsnet,v2.8,immunization,imm_route_concept_id,No,concept,concept_id,A foreign key that refers to a standard administration route concept identifier in the Vocabulary.
 pedsnet,v2.8,immunization,imm_route_source_value,No,,,The information about the route of administration as detailed in the source
+pedsnet,v2.8,immunization,visit_occurrence_id,No,,,A foreign key that refers to the visit associated with the immunization record.
+pedsnet,v2.8,immunization,procedure_occurrence_id,No,,,A foreign key that refers to the visit associated with the immunization record.
```
Comment (murphyke) [inline pedsnet/v2.8/definitions/immunization.csv]: s/visit/procedure/ in the description

**Comment by burrowse** [comment]
Hi @murphyke 

Thanks! I made that change!

Sure. I can give re-forking a try next time. I attempted to update my master branch to reflect the data-models master branch and made the change on a feature branch `update-immunization` but it seems like that still didn't work.

**Comment by murphyke** [comment]
OK re: re-forking. Or you can clean up your master by doing:

```
# If you haven't done this yet ...
git remote add upstream https://github.com/chop-dbhi/data-models
# Then ...
git checkout SOME_BRANCH_BESIDES_MASTER
git branch -d master
git checkout -b master upstream/master
```

**Comment by murphyke** [comment]
LGTM, merging.

**Pull request merged**